js/messages.js

```diff
@@ -31,7 +31,9 @@ function renderMessages(message_data) {
     if (message["Twitter"]) {
       let userSocial = document.createElement("a");
       userSocial.href = `https://twitter.com/${message["Twitter"]}`;
+      userSocial.setAttribute('target', '_blank');
       userSocial.textContent = message["Twitter"];
+      userSocial.title = "Twitter page for " + message["Participant Name"] + " (" + message["Twitter"] + ")";
```

Comment:
you could delete these - titles are actually not beneficial to accessibility since they're only activated with mouse and do nothing for keyboard/ screenreader